Source/UI/Test Runner/IPFTestRunnerViewController.swift

```diff
@@ -0,0 +1,247 @@
+import AudioToolbox
+import UIKit
+
+private enum IPFTestState {
+    case active
+    case inactive
+}
+
+final class IPFTestRunnerViewController: FormViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        restoreTestSettings()
+        setupUI()
+        updateUI(state: .inactive)
+    }
+
+    private var configuration = IPFTestRunnerConfiguration()
+    private var testRunner: IPFTestRunner?
+    private var testLocation = ""
+    private let durations: [UInt] = [5, 10, 30, 300]
```

Comment:
We can remove the 5 second test if it doesn't make sense. It mostly helped me to run a lot of tests quickly